File: src/test/java/cernx/colt/matrix/tdouble/impl/DenseColumnDoubleMatrix2DViewTest.java
package cernx.colt.matrix.tdouble.impl;

public class DenseColumnDoubleMatrix2DViewTest extends DenseColumnDoubleMatrix2DTest {

    public DenseColumnDoubleMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new DenseColumnDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new DenseColumnDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new DenseColumnDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}

File: src/test/scala/cern/colt/matrix/tdouble/impl/DenseColumnDoubleMatrix2DViewTest.scala
package cern.colt.matrix.tdouble.impl

import cern.colt.matrix.MatrixTypes.DenseColumnDoubleMatrix2D

class DenseColumnDoubleMatrix2DViewTest(arg0: String) extends DenseColumnDoubleMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new DenseColumnDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new DenseColumnDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new DenseColumnDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}
